dev-cpp/notcurses/notcurses-2.2.3.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS="cmake"
+inherit cmake-multilib
+
+DESCRIPTION="Blingful TUIs and character graphics"
+HOMEPAGE="https://notcurses.com"
+SRC_URI="https://github.com/dankamongmen/notcurses/archive/v${PV}.tar.gz -> ${P}.tar.gz https://github.com/dankamongmen/notcurses/releases/download/v${PV}/notcurses-doc-${PV}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
```

Comment:
Without me trying this new ebuild, what does `doc` do to the ebuild now? How do you install the downloaded extra documentation?

Reply:
as i'm envisioning things, `doc` would cover installation of the man pages from the doc package (but not download of said package). that's what i intend to implement. i'm also happy to unconditionally install the man pages.

Reply:
by the way, you mention installing .xhtml files somewhere, but i'm not intended to do that. the only files i'd copy out of the doc tarball would be the man pages. the .xhtml files have a bijection on the man pages, and aren't really useful IMHO on a development workstation (they're intended more for a web server).

Reply:
so what i'm planning to do is: add a `src_install` function, which will run `cmake-multilib_src_install`, and use the `doman` command to install the man pages, iff the `doc` variable is set. does that sound about right?

Reply:
I've got this all written out now, and just need to test with `doc` in place. I'm not certain that I'm unpacking the doc tarball to a place where `doman` is going to find it. Everything seems to work in the no-`doc` case.